### PR TITLE
Added logos and icons

### PR DESCRIPTION
**Pull request by deepesh16b**
@22shubh22 @iamakhileshmishra 
I have added icons, logos and changed dark color
Screenshot 2022-10-08 122950

### CODE REVIEW

**Comment by deepesh16b** [comment]
@iamakhileshmishra @22shubh22 
Please review

File: index.html [hunk 1]
```diff
@@ -33,16 +33,21 @@
...
+            <input type="checkbox" id="dark-theme-toggler" />
+            <span class="slider round"></span>
+          </label>
+      </div>
+      <img class="moon-icon" src="images/moon.png" alt="">
     </div>
     <!-- Live Date & Time -->
     <span id="span" class="d-flex align-items-center justify-content-center">Loading.....</span>
     <hr />
+    <img class="contest-img" src="images/leetcode.png" alt="Leetcode image">
```
Comment (22shubh22) [inline index.html]: contest-img should not be outside the contests div. 

This will make leetcode icon appear irrespective of the contests

**Review by 22shubh22 — CHANGES_REQUESTED** [review]
@deepesh16b

File: index.html [hunk 2]
```diff
@@ -47,10 +47,10 @@
     <!-- Live Date & Time -->
     <span id="span" class="d-flex align-items-center justify-content-center">Loading.....</span>
     <hr />
-    <img class="contest-img" src="images/leetcode.png" alt="Leetcode image">
     <div id="contests" class="d-flex flex-column container">
       <div class="card mb-2">
         <div class="card-body">
+          <img class="contest-img" src="images/leetcode.png" alt="Leetcode image">
```
Comment (22shubh22) [inline index.html:53]: @deepesh16b Nice.
Now, can you programmatically change the icon based on the contest?

For now, whether it is any contest site, leetcode ICON will be displayed.

Comment (deepesh16b) [reply index.html:53]: Sorry I don't know how to do that, i am a newbie

Comment (22shubh22) [reply index.html:53]: okay.

**Pull request closed without merging**